### ISSUE TITLE
Assign ticket to equipment via email

### ISSUE THREAD

**Issue by DaAntidote** [poctob/OSTEquipmentPlugin#7]
Enhancement; If there is an Asset ID variable then it might feasible for a ticket sent in via email to be auto associated to an asset..

Althernatly; is there a way tickets can be auto associated to asset ID via email?

**Comment by bradleyh** [comment]
Any news on this? Did it ever get implemented? 

Thanks,

**Comment by poctob** [comment]
No.  Unfortunately there is no plans for this functionality, at least not until OSTicket team improves plugin and API support.

**Comment by bradleyh** [comment]
Hi,  

Thanks for getting back to me, no problem.

Perhaps it would suffice if when a staff member updates a device state in a ticket it saves and replicates on the front-end?

Currently it does not do this (we'd like to be able to flag a ticket when they come in via email - bypasses the need for an auto apply on import. )

Thanks!

**Comment by poctob** [comment]
Not sure what you mean here.  Are you saying that when you change an equipment status in the back-end, it doesn't show correct status in the front-end?

**Comment by bradleyh** [comment]
Indeed. It doesn't update on the front

**Comment by poctob** [comment]
Can you file a new issue for this problem? Please include as much info as possible, screenshots would be helpful too.

Thanks!
